tests/oauth2/OAuth2SummitSponsorApiTest.php

```
@@ -1656,4 +1656,75 @@ public function testGetAllSponsorsBySummitPublic(){
+            'id' => self::$summit->getId(),
+        ];
+
+        // forms_qty is a string instead of integer — should fail validation
+        $data = [
+            [
+                'sponsor_id' => self::$sponsors[0]->getId(),
+                'forms_qty'  => 'not-an-integer',
+            ],
+        ];
+
+        $response = $this->action(
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$response'. (undefined)

(UnusedLocalVariable)